File: 0070 climbing-stairs/solution.cpp
class Solution {
public:
    int climbStairs(int n) {
        if(n <=3) return n;
        int f=2,s=3;
        for(int i=4;i<=n;i++) {
            int t = f+s;
            f=s;
            s=t;
        }
        return s;
    }
};

File: 0070 climbing-stairs/solution.rs
impl Solution {
    pub fn climb_stairs(n: i32) -> i32 {
        if n <=3 {
            return n
        } 
        let mut f = 2;
        let mut s = 3;
        for i in 4..=n {
            let t = f+s;
            f=s;
            s=t;
        }
        s
    }
}
